src/main/drivers/vtx_common.h

```diff
@@ -28,21 +28,25 @@ typedef enum {
 
 struct vtxVTable_s;
 
+typedef struct vtxDeviceCapability_s {
+    uint8_t bandCount;
+    uint8_t channelCount;
+    uint8_t powerCount;
+} vtxDeviceCapability_t;
+
 typedef struct vtxDevice_s {
     const struct vtxVTable_s *vTable;
 
-    uint8_t numBand;
-    uint8_t numChan;
-    uint8_t numPower;
+    vtxDeviceCapability_t capability;
 
-    uint16_t *freqTable;  // Array of [numBand][numChan]
-    char **bandNames;    // char *bandNames[numBand]
```

Comment:
@jflyper agreed.  Also the presence of a RTC6705_POWER_PIN has an impact on the powerNames too.  In the case of the NEO it's OFF, 25, 200, on the sirinfpv it's 25/200.  It also depends on the power amplifiers used, it it might be 200/600 on some boards, and that's only known because of the target configuration.  The mW values can be arbitrary too, so there's no point coming up with an enum trying to name them all and having a mapping and string to id mappings in the higher level code that needs strings.